packages/devtools_app/test/inspector/inspector_screen_test.dart

```diff
@@ -4,16 +4,10 @@
 
 import 'dart:convert';
 
-import 'package:devtools_app/src/config_specific/ide_theme/ide_theme.dart';
-import 'package:devtools_app/src/screens/inspector/diagnostics_node.dart';
-import 'package:devtools_app/src/screens/inspector/inspector_screen.dart';
-import 'package:devtools_app/src/screens/inspector/inspector_tree.dart';
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
import libraries directly instead of importing devtools_app.dart

Reply:
the auto import on these always gets me :face-palm:
thank you for noticing these. I will be more diligent moving forwards!